RoutersTest.test_service_tags is not stable

https://api.travis-ci.org/v3/job/455918988/log.txt

[ERROR]   RoutersTest.test_service_tags:217 expected: <0.6> but was: <0.87>

```
I 1116-1059:19,614 i.s.s.BaseTest ***** Test started  : RoutersTest.test_remote_service_tags() ***** [main]
Service B was called: 76 times.
I 1116-1059:20,677 i.s.s.BaseTest ***** Test finished : RoutersTest.test_remote_service_tags() ***** [main]
[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.976 s <<< FAILURE! - in io.scalecube.services.routings.RoutersTest
[ERROR] test_service_tags  Time elapsed: 3.136 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <0.6> but was: <0.87>
	at io.scalecube.services.routings.RoutersTest.test_service_tags(RoutersTest.java:217)
```

--- a/services/src/test/java/io/scalecube/services/routings/RoutersTest.java
+++ b/services/src/test/java/io/scalecube/services/routings/RoutersTest.java
@@ -29,7 +29,6 @@ import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
@@ -56,7 +55,7 @@ public class RoutersTest extends BaseTest {
                     .tag("SENDER", "1")
                     .build(),
                 ServiceInfo.fromServiceInstance(new GreetingServiceImplA())
-                    .tag("Weight", "0.3")
+                    .tag("Weight", "0.1")
                     .build())
             .startAwait();
 
@@ -70,7 +69,7 @@ public class RoutersTest extends BaseTest {
                     .tag("SENDER", "2")
                     .build(),
                 ServiceInfo.fromServiceInstance(new GreetingServiceImplB())
-                    .tag("Weight", "0.7")
+                    .tag("Weight", "0.9")
                     .build())
             .startAwait();
   }
@@ -111,7 +110,6 @@ public class RoutersTest extends BaseTest {
     assertTrue(!result1.sender().equals(result2.sender()));
   }
 
-  @Disabled("https://github.com/scalecube/scalecube-services/issues/439")
   @Test
   public void test_remote_service_tags() throws Exception {
 
@@ -126,7 +124,7 @@ public class RoutersTest extends BaseTest {
 
     AtomicInteger serviceBCount = new AtomicInteger(0);
 
-    int n = (int) 1e2;
+    int n = (int) 1e3;
 
     Flux.range(0, n)
         .flatMap(i -> service.greeting(new GreetingRequest("joe")))
@@ -134,9 +132,11 @@ public class RoutersTest extends BaseTest {
         .doOnNext(response -> serviceBCount.incrementAndGet())
         .blockLast(Duration.ofSeconds(3));
 
-    System.out.println("Service B was called: " + serviceBCount.get() + " times.");
+    System.out.println("Service B was called: " + serviceBCount.get() + " times out of " + n);
 
-    assertEquals(0.6d, serviceBCount.doubleValue() / n, 0.25d);
+    assertTrue(
+        (serviceBCount.doubleValue() / n) > 0.5,
+        "Service B's Weight=0.9; more than half of invocations have to be routed to Service B");
   }
 
   @Test
@@ -192,7 +192,6 @@ public class RoutersTest extends BaseTest {
     }
   }
 
-  @Disabled("https://github.com/scalecube/scalecube-services/issues/439")
   @Test
   public void test_service_tags() throws Exception {
 
@@ -207,7 +206,7 @@ public class RoutersTest extends BaseTest {
 
     AtomicInteger serviceBCount = new AtomicInteger(0);
 
-    int n = (int) 1e2;
+    int n = (int) 1e3;
     for (int i = 0; i < n; i++) {
       ServiceMessage message = service.requestOne(req, GreetingResponse.class).block(timeout);
       if (message.data().toString().contains("SERVICE_B_TALKING")) {
@@ -215,8 +214,11 @@ public class RoutersTest extends BaseTest {
       }
     }
 
-    System.out.println("Service B was called: " + serviceBCount.get() + " times.");
+    System.out.println("Service B was called: " + serviceBCount.get() + " times out of " + n);
 
-    assertEquals(0.6d, serviceBCount.doubleValue() / n, 0.25d);
+    assertTrue(
+        (serviceBCount.doubleValue() / n) > 0.5,
+        "Service B's Weight=0.9; at least more than half "
+        + "of invocations have to be routed to Service B");
   }
 }

